### PR TITLE
Add two features `calculateReadTime` and `slugify`.

### PR DESCRIPTION
**Pull request by jeonghoyang12**
1. calculateReadTime
- Add the feature that calculates the read time based on the number of characters in the content.

2. slugify
- Add the feature that makes slug based on the title.

### CODE REVIEW

File: src/utils/slugify.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @returns The slugified string
+ */
+
+export function slugify(str: string): string {
+  return str
+    .toLowerCase()
+    .trim()
+    .replace(/[^\w\s-]/g, '') // Remove non-word characters
+    .replace(/[\s_-]+/g, '-') // Replace spaces, underscores, and consecutive dashes with a single dash
+    .replace(/^-+|-+$/g, '') // Remove leading and trailing dashes
```
Comment (devjiwonchoi) [inline src/utils/slugify.ts:15]: Can the first & last regex be merged?

Comment (jeonghoyang12) [reply src/utils/slugify.ts:15]: Can be merged, but handled these two separately for readability. Want me to merge them?

Comment (devjiwonchoi) [reply src/utils/slugify.ts:15]: Nah we can leave them no worries.
You wanna add test or want me to add one for you?

Comment (jeonghoyang12) [reply src/utils/slugify.ts:15]: Will be grateful if you add one for me.

Comment (devjiwonchoi) [reply src/utils/slugify.ts:15]: Cool. Will add one on the following PR. Thanks!

**Review by devjiwonchoi — APPROVED** [review]

**Pull request merged**